Launcher/Classes/ViewController.swift

```diff
@@ -434,6 +421,10 @@ class TasksViewController: NSViewController {
             arguments.append("-p \(cucumberProfile)")
         }
         
+        if !newOne.stringValue.isEmpty {
```

Comment:
Can you please check what this `newOne` is supposed to do and change its name? :)